layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -665,6 +665,204 @@ bool CoreChecks::ValidateAccelerationBuffers(VkCommandBuffer cmd_buffer, uint32_
                 }
                 break;
             }
+            case VK_GEOMETRY_TYPE_SPHERES_NV: {
+                const Location p_geom_geom_spheres_loc = p_geom_geom_loc.pNext(Struct::VkAccelerationStructureGeometrySpheresDataNV);
+                auto sphere_struct = reinterpret_cast<VkAccelerationStructureGeometrySpheresDataNV const *>(geom_data.pNext);
+                ASSERT_AND_RETURN_SKIP(sphere_struct);
+                // Check the validity of buffer references for vertex, index, and radius data
+                skip |= buffer_check(geom_i, sphere_struct->vertexData, p_geom_geom_spheres_loc.dot(Field::vertexData));
+                skip |= buffer_check(geom_i, sphere_struct->indexData, p_geom_geom_spheres_loc.dot(Field::indexData));
+                skip |= buffer_check(geom_i, sphere_struct->radiusData, p_geom_geom_spheres_loc.dot(Field::radiusData));
+
+                // Check the validity of the vertex radius and index data
+                if (!sphere_struct->vertexData.deviceAddress && !sphere_struct->vertexData.hostAddress) {
+                    skip |= LogError("VUID-VkAccelerationStructureGeometrySpheresDataNV-vertexData-10430", cmd_buffer,
+                                     p_geom_geom_spheres_loc.dot(Field::vertexData),
+                                     "The memory address in vertexData must not be 0 or `NULL'");
```

Comment:
```suggestion
                                     "must not be 0 or NULL");
```

same below for `radiusData`